pkgs/development/libraries/clfft/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, cmake, fftw, fftwFloat, boost166, opencl-clhpp, ocl-icd }:
+
+let
+  version = "2.12.2";
+in stdenv.mkDerivation {
+  pname = "clfft";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "clMathLibraries";
+    repo = "clFFT";
+    rev = "ce107c4d5432d70321af8980a5e7fb64c4b4cce4";
```

Comment:
For clarity we prefer to fetch the `v${version}` tag from upstream

Reply:
okay. this is the commit from that version, but i can surely see why that's preferable. fixing.